pkgs/development/python-modules/jupyter-repo2docker/default.nix

```diff
@@ -0,0 +1,38 @@
+{ stdenv
+, buildPythonPackage
+, fetchPypi
+, pkgs
```

Comment:
Don't pass in `pkgs`, pass in individual deps

Reply:
So the reason that I did this is that the package is `docker` that is a name collision with `pythonPackages.docker`. Perfect time to ask how should this be handled? How is this?

```
  jupyter-repo2docker = callPackage ../development/python-modules/jupyter-repo2docker {
    docker = pkgs.docker;
    python-docker = docker;
  };
```

Reply:
`callPackage` tries to fill in the parameters using the given scope. That means within `python-packages.nix` it will pick attributes from that file/scope. Therefore, your function (`jupyter-repo2docker`) needs 
For expressions in `python-packages.nix`, the function parameters should match the attributes defined there. So something like
```
{ ...
, docker
, pkgs-docker

}: buildPythonPackage { ... }
```
and
```
jupyter-repo2docker = callPackage ... {
  pkgs-docker = pkgs.docker;
}
```

Reply:
Thanks for the advice. I have made the change and also forgot that this package only builds for python >= 3.4. So I have disabled the package if less than 3.4.